web/src/views/SubmissionPortal/Components/TemplateChooser.vue

```diff
@@ -32,11 +32,11 @@ export default defineComponent({
 <template>
   <div>
     <div class="text-h2">
-      Environmental Extensions
+      Sample Environment
       <submission-docs-link anchor="environmental-package" />
     </div>
     <div class="text-h5">
-      Choose environmental extensions for your data.
+      Choose the MIxS Extension for your samples.
```

Comment:
If you want to link the "MIxS Extension" part it would look like this. The line breaks make it look a little funky, but that's what the code format-checker insists on.

```suggestion
      Choose the
      <a
        href="URL GOES HERE"
        target="_blank"
        rel="noopener noreferrer"
      >MIxS Extension</a>
      for your samples.
```

Reply:
Sorry! I somehow missed this but @pkalita-lbl  is correct.